Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/MergerWindow.xaml

```diff
@@ -0,0 +1,56 @@
+﻿<Page x:Class="Ginger.ApplicationModelsLib.APIModels.APIModelWizard.MergerPage"
```

Comment:
please change the page class + file name to be "APIModelsCompareMergePage" (instead of current "MergerPage" / "MergerWindow")

Reply:
Done !